barretenberg/cpp/src/barretenberg/vm/tests/avm_cast.test.cpp

```diff
@@ -23,6 +23,7 @@ class AvmCastTests : public ::testing::Test {
 
     VmPublicInputs public_inputs;
     AvmTraceBuilder trace_builder;
+    std::vector<FF> calldata{};
```

Comment:
`{}` not needed in any `std::vector<T> xxx;` (unlike array, AFAIK, this does not zero-initialize)